rmgpy/qm/molecule.py

```diff
@@ -409,18 +409,25 @@ def parse(self):
         parser.logger.setLevel(
             logging.ERROR
         )  # cf. http://cclib.sourceforge.net/wiki/index.php/Using_cclib#Additional_information
-        parser.rotcons = (
-            []
-        )  # give it an attribute and it won't delete it, leaving it on the parser object
+
```

Comment:
I think this needs to be left in for backwards compatibility with cclib <1.8.0.

Here is my recommendation for lines 412-426:
```suggestion
parser.molmass = None # give it an attribute and it won't delete it, leaving it on the parser object
parser.rotcons = (  # for cclib < 1.8.0
    []
)
parser.rotconsts = (  # for cclib >= 1.8.0
    []
)
cclib_data = parser.parse()  # fills in either parser.rotcons or parser.rotconsts but not both
assert bool(parser.rotconsts) != bool(parser.rotcons)
if parser.rotcons:  # for cclib < 1.8.0
    cclib_data.rotcons = (
        parser.rotcons
    )
else:  # for cclib >= 1.8.0
    cclib_data.rotconsts = (
        parser.rotconsts
    )
```

Reply:
So in total, I think the parse function in RMG-Py/rmgpy/qm/molecule.py should look like this:
```
def parse(self):
        """
        Parses the results of the Mopac calculation, and returns a QMData object.
        """
        parser = self.get_parser(self.output_file_path)
        parser.logger.setLevel(
            logging.ERROR
        )  # cf. http://cclib.sourceforge.net/wiki/index.php/Using_cclib#Additional_information

        parser.molmass = None  # give it an attribute and it won't delete it, leaving it on the parser object
        parser.rotcons = (  # for cclib < 1.8.0
            []
        )
        parser.rotconsts = (  # for cclib >= 1.8.0
            []
        )
        cclib_data = parser.parse()  # fills in either parser.rotcons or parser.rotconsts but not both
        assert bool(parser.rotconsts) != bool(parser.rotcons)
        if parser.rotcons:  # for cclib < 1.8.0
            cclib_data.rotcons = (
                parser.rotcons
            )
        else:  # for cclib >= 1.8.0
            cclib_data.rotconsts = (
                parser.rotconsts
            )

        radical_number = self.molecule.get_radical_count()
        cclib_data.molmass = (
            parser.molmass
        )  # this hack required because molmass is not part of a default cclib data object
        qm_data = parse_cclib_data(
            cclib_data, radical_number + 1
        )  # Should `radical_number+1` be `self.molecule.multiplicity` in the next line of code? It's the electronic ground state degeneracy.
        return qm_data
```